cmds/shutdown/shutdown.go

```diff
@@ -17,18 +17,18 @@ package main
 
 import (
 	"flag"
+	"golang.org/x/sys/unix"
```

Comment:
I think this should go after the standard library imports, at least that's the order `gofmt` produces.

Reply:
@tklauser Thank you, addressed via a5e7783.